spec/features/user_adds_coverage_to_a_course_spec.rb

```diff
@@ -22,9 +22,59 @@
     end
   end
 
+  scenario "user adds additional coverage to a course
+  from the coverages page by clicking on an unmatched outcome" do
+    unmatched_outcome = create(:outcome,
+                              nickname: "Astrophysics",
+                              label: "AP",
+                              description: "an ability to do Astrophysics well")
+    outcome = create(:outcome,
+                    nickname: "Ecology",
+                    label: "EE",
+                    description: "an ability to do Ecology well")
+    unselected_subject = create(:subject)
+    course = outcome.course
+    coverage = create(:coverage, course: course, outcomes: [outcome])
+    coverage.course.outcomes << unmatched_outcome # associate the unmatched_outcome with a course, but not a subject
+    user = user_with_assessments_access_to(course.department)
+
+    visit manage_assessments_course_path(course.id, as: user)
+    click_on unmatched_outcome.nickname
+
+    expect(page).to have_header("Add a class")
+    expect(page).to have_prepopulated_select_box(unmatched_outcome.nickname)
```

Comment:
I think this is really all we care about in this test case. We're reasonably certain the form work because we tested it in the test case above. I think we can end this test case here. What do you think?